src/libraries/Native/AnyOS/System.Globalization.Native/configure.cmake

```diff
@@ -0,0 +1,34 @@
+include(CheckCSourceCompiles)
```

Comment:
I don't know why this file wasn't marked as renamed as I used git mv to move all the files. But this file contains a few edits from the original one.